components/support/sync-telemetry/pings.yaml

```diff
@@ -4,6 +4,17 @@
 
 $schema: moz://mozilla.org/schemas/glean/pings/1-0-0
 
+sync:
+  description: >
+    A summary sync ping, sent for every sync.
```

Comment:
Since this will end up in the autogenerated docs, it might be worth clarifying a bit to give more context to people who do not know much about how sync works.

This says `sent for every sync`, but other pings say, for example,  `sent for every history sync`. Is there any way to phrase this so that disambiguates between the "engine sync" and the "global sync"?

Please ignore this if you feel like that's already clear.

Reply:
Updated the description, thanks for pointing this out.